facia-tool/app/tools/FaciaToolArchive.scala

```diff
@@ -0,0 +1,54 @@
+package tools
+
+import com.amazonaws.regions.{Regions, Region}
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsyncClient
+import com.amazonaws.services.dynamodbv2.model._
+import common.{ExecutionContexts, Logging}
+import conf.{Switches, Configuration}
+import org.joda.time.{DateTimeZone, DateTime}
+import play.api.libs.json._
+import scala.collection.JavaConverters._
+import awswrappers.dynamodb._
+
+import scala.util.{Failure, Success}
+
+case class ArchiveRequest(email: String, updateJson: JsValue, diff: JsValue)
+
+object FaciaToolArchive extends ExecutionContexts with Logging {
+  val TableName = "FaciaToolUpdateHistory"
+  private val dynamoClient: Option[AmazonDynamoDBAsyncClient] =
+    if (Configuration.environment.isProd) {
+      val c = new AmazonDynamoDBAsyncClient()
+      c.setRegion(Region.getRegion(Regions.EU_WEST_1))
+      Option(c)
+    } else None
+
+  def dayKey(date: DateTime) = date.toString("yyyy-MM-dd")
```

Comment:
@robertberry I reckon the latency is fast enough. Check out the current latency for our current streams. It's spikes you would have to be worried about.

Yeah, probably a better idea to go through `dynamo`, *especially* since updates to a dynamo table can automatically feed a stream of updates!

What don't they do these days? I reckon amazon are pushing for the one-button-stack, like an iphone. :8ball: